runner/webserver.ts

```diff
@@ -142,12 +143,13 @@ export function webserver(wct: Context): void {
     // webserver as they please.
     await wct.emitHook('prepare:webserver', app);
 
-    // Serve up all the static assets.
-    app.use(serveWaterfall(wsOptions.pathMappings, {
+    // Serve up project & dependencies via polyserve
+    const polyserve = makeApp({
       root: options.root,
       headers: DEFAULT_HEADERS,
-      log: wct.emit.bind(wct, 'log:debug'),
-    }));
+      packageName: path.basename(options.root),
+    });
+    app.use('/components/', polyserve);
```

Comment:
Check out how polyserve uses `makeApp()`: https://github.com/PolymerLabs/polyserve/blob/master/src/start_server.ts#L146 `makeApp()` returns a router that handles all routes on the `components/` path. This lets us continue to control the rest of the server that wct needs to manage. 

I've added as a todo to investigate what it would look like for polyserve to return a full express application, and for wct to build off of that. If possible, this would be the first step towards that.